stage1/init/init.go

```diff
@@ -137,6 +137,7 @@ func parseFlags() *stage1commontypes.RuntimePod {
 
 	flag.Parse()
 
+	rp.Debug = debug
```

Comment:
Unfortunately, duplicating seems safer at this point. We have multiple different `debug` globals around the codebase, switching the flag parser will probably introduce subtle misbehavior.